cypress-io/cypress
# fix: change wording for spec file creation card

- Closes [issue-21908](https://github.com/cypress-io/cypress/issues/21908)

### User-facing changelog

Fix misleading wording when creating a new starter spec file in the UI.

### Additional details

N/A

### Steps to test

- Launch Cypress's test runner.
- Click on the button `New Spec` button.
- You should see the new wording in the cards to create a new starter spec file.

### How has the user experience changed?

**Before changes:**

<img width="813" alt="Screenshot 2022-11-26 at 6 06 16 AM" src="https://user-images.githubusercontent.com/32891808/204085736-d7b53a19-dd0a-4957-a36e-688850d8e283.png">

**After changes:**

<img width="736" alt="Screenshot 2022-11-26 at 6 11 22 AM" src="https://user-images.githubusercontent.com/32891808/204085765-7f565105-c916-4ca2-858c-83bd9730d296.png">

### PR Tasks
